nautobot_ssot/integrations/aristacv/signals.py

```diff
@@ -120,9 +120,10 @@ def post_migrate_create_platform(apps=global_apps, **kwargs):
     """Callback function for post_migrate() -- create Arista Platform."""
     Platform = apps.get_model("dcim", "Platform")
     Manufacturer = apps.get_model("dcim", "Manufacturer")
-    Platform.objects.get_or_create(
+    Platform.objects.update_or_create(
         name="arista.eos.eos",
         napalm_driver="eos",
+        network_driver="arista_eos",
         manufacturer=Manufacturer.objects.get(name="Arista"),
     )
```

Comment:
```suggestion
    Platform.objects.update_or_create(
        name="arista.eos.eos",
        napalm_driver="eos",
        network_driver="arista_eos",
        manufacturer=Manufacturer.objects.get(name="Arista"),
    )
```
```suggestion
    Platform.objects.update_or_create(
        name="arista.eos.eos",
        defaults={
        "napalm_driver": "eos",
        "network_driver": "arista_eos",
        "manufacturer": Manufacturer.objects.get(name="Arista"),
        }
    )
```

Otherwise the updating will never happen and I believe it might crash when it encounters a platform called `arista.eos.eos` that doesn't have the other parameters matching

Reply:
Ahhh, wasn't aware of that. Good catch!